src/dpp/guild.cpp

```diff
@@ -347,6 +347,11 @@ bool guild_member::has_rejoined() const {
 	return flags & dpp::gm_did_rejoin;
 }
 
+bool guild_member::is_guild_owner() const {
+	auto* _guild = find_guild(user_id);
```

Comment:
this is wrong. you're using user id to search for a guild

Reply:
Yeah I might have messed it up, will fix